packages/auth/package.json

```
@@ -7,7 +7,7 @@
   ],
   "scripts": {
     "knex": "knex",
-    "build:deps": "pnpm --filter open-payments build",
+    "build:deps": "pnpm --filter open-payments build && pnpm --filter http-signature-utils build",
```

```suggestion
    "build:deps": "pnpm --filter open-payments build",
```
`open-payments` builds `http-signature-utils`